Source/WebCore/css/ShorthandSerializer.cpp

```diff
@@ -1315,6 +1318,52 @@ String ShorthandSerializer::serializeTextWrap() const
     return makeString(nameLiteral(mode), ' ', nameLiteral(style));
 }
 
+String ShorthandSerializer::serializeAnimationRange() const
+{
+    auto& rangeStart = longhandValue(0);
+    auto& rangeEnd = longhandValue(1);
+
+    auto rangeStartID = rangeStart.valueID();
+
+    auto rangeStartValue = emptyString();
```

Comment:
Not great to have a local variable named `rangeStart` containing the range start value and a second one named `rangeStartValue`, since the names make it unclear how they are related. I would use the names `start` and `startString` or `start` and `serializedStart`.

Reply:
Fixed.